index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav class="navigation">
+            <a class="logo" href="#">Hami</a>
+            <img class="humburger" src="images/Union.svg" alt="">
+        </nav>
+    </header>
+    <main>
+        <h1 class="hero">I’m Hamdullah
+            Glad to see you!
+        </h1>
+        <p class="hero-details">I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+
+        <p class="connect">LET'S CONNECT</p>
+
+        <ul class="link">
+            <li class="social-links"><a href="https://twitter.com"><img src="images/twitter.svg" alt="twitter"></a></li>
+            <li class="social-links"><a href="https://linkedin.com"><img src="images/Linkedin.svg" alt="linkedin"></a></li>
+            <li class="social-links"><a href="https://mail.google.com"><img src="images/gmail.svg" alt="gmail"></a></li>
+            <li class="social-links"><a href="https://github.com"><img src="images/github.svg" alt="github"></a></li>
+            <li class="social-links"><a href="https://vectorstock.com"><img src="images/vector.svg" alt="vector"></a></li>
+        </ul>
```

Comment:
- [ ] Kindly consider putting your own social media links in the `a` tag's `href` attribute so a visitor can see your professional work. you can omit that if you don't have a account yet for example on `vertor stock`.
  -  Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.